### DISCUSSION TITLE
API Usage

### DISCUSSION THREAD

**Discussion by nicoaudy** [forge-42/base-stack#17] [Q&A]
Hi @AlemTuzlak thanks for making this repo, it's interesting for me since the server is using Hono. I have a question how create an API endpoint on the Hono side or just create an API folder on the remix folder? Do you know how to use this?

**Comment by AlemTuzlak** [comment]
Both is fine, if you're just creating endpoitns for external consumption you can go with hono, if you're creating something to be consumed by your remix app + external you can use resource routes, it's really up to you
